src/SoundList.js

```diff
@@ -32,13 +32,23 @@ const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
                   {track.username}
                 </span>
               </div>
-              <div className="hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0 space-x-12">
-                <IonIcon className="pr-4" icon={timeSharp} size="large" />
-                {timeConvert(track.duration)}
-                <IonIcon className="pr-4" icon={downloadSharp} size="large" />
-                {track.num_downloads}
-                <IonIcon className="pr-4" icon={playCircleSharp} size="large" />
-                {track.num_ratings}
+              <div className="w-6/12 hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0">
+                <div className="flex justify-start w-2/12 items-center">
```

Comment:
Change `w-2/12` to `w-4/12`

Reply:
I tried that but then they're too far apart, I am trying to make them to look as close to the Figma as possible.